extensions/BMO/template/en/default/account/create.html.tmpl

```diff
@@ -83,31 +83,19 @@ function onSubmit() {
   [% INCLUDE product
       icon = "firefox"
       name = "Firefox Support"
-      url = "http://support.mozilla.com/"
+      url = "https://support.mozilla.org/"
       desc = "Support for the Firefox web browser."
   %]
   [% INCLUDE product
-      icon = "firefox"
-      name = "Firefox for Mobile Support"
-      url = "http://support.mozilla.com/mobile"
-      desc = "Support for the Firefox Mobile web browser."
-  %]
-  [% INCLUDE product
       icon = "thunderbird"
       name = "Thunderbird Support"
-      url = "http://www.mozillamessaging.com/support/"
+      url = "https://support.mozilla.org/products/thunderbird"
       desc = "Support for Thunderbird email client."
   %]
   [% INCLUDE product
-      icon = "other"
-      name = "Support for other products"
-      url = "http://www.mozilla.org/projects/"
-      desc = "Support for products not listed here."
-  %]
-  [% INCLUDE product
       icon = "input"
       name = "Feedback"
-      url = "http://input.mozilla.com/feedback"
+      url = "https://input.mozilla.org/feedback"
```

Comment:
Is this still actually operational and in use?

Reply:
AFAIK input.m.o is no longer developed, but it's now redirected to a new feedback page.